app/components/ui/Section/CategorySection.module.scss

```diff
@@ -0,0 +1,36 @@
+@import "../../../styles/utils/_helpers.scss";
+
+.categorySection {
```

Comment:
Not sure if I missed this the first time but make sure this section has the same max-content width as the other sections. The cards should be vertically aligned with the content above and below.

Also, the section title and description do not match the mockups if you could fix that too

<img width="2256" alt="image" src="https://github.com/Exygy/askdarcel-web/assets/60076088/b87ddece-8c3c-47ad-a88a-54ec32fbc3a5">

Reply:
I think this should utilize a reusable component i.e. HomePageSection and just pass in the cards section as children

Reply:
Thanks for pointing that out, all of these issues should be addressed in the [home page PR](https://github.com/Exygy/askdarcel-web/pull/37) where I have the updated HomePageSection component. I will make a change where I'll delete the title and description from this PR